### ISSUE TITLE
Create dev deployment for crawler

### ISSUE THREAD

**Issue by jeffmcaffer** [clearlydefined/crawler#4]
The crawler needs a deployment with an authtoken that can be used ot secure calls.

When this is done, configure it into the dev CD service by setting the following application settings

* CRAWLER_AUTH_TOKEN
* CRAWLER_API_URL

**Comment by jeffmcaffer** [comment]
I created a deployment for this but there are several issues:

* The docker image for the Crawler does not build. It uses nodegit (NPM) that has some interesting install/build issues. Oddly worked fine on my windows machine but not so much on alpine during docker build. Docker on my machine is not so functional so hopefully someone else can drive in on that. There were some related issues e.g. https://github.com/nodegit/nodegit/issues/1039

* The Linux App service I created seems to not allow the setting of a container/registry. Looks like there might be some sort of manual step needed 

```
az webapp config container set --name <app_name> --resource-group myResourceGroup --docker-custom-image-name mydockerimage --docker-registry-server-url https://<azure-container-registry-name>.azurecr.io --docker-registry-server-user <registry-username> --docker-registry-server-password <password>
```
scroll all the way to the bottom of https://docs.microsoft.com/en-us/azure/app-service/containers/tutorial-custom-docker-image 

I could be missing something... In any event, that is sure to fail until there is actually an image in that registry and that is dependent on the build issue.
